lib/coverband/utils/s3_report_writer.rb

```diff
@@ -0,0 +1,59 @@
+# frozen_string_literal: true
+
+module Coverband
```

Comment:
It looks like we are still writing scov report to s3 and reporting from their? Would the idea be to have a rake task run periodically that takes data from redis, generates a report, and writes report to s3 as we do for coverband2?

Reply:
Yeah so I want to make the web version able to just read from the store and render the HTML, but given some of the issues on 2.x I am not sure I will get that into the initial 3.0.0 release but it could likely be added during the 3.0.0 lifecycle without breaking changes.

I don't really use the `rake coverband:coverage` task anymore and mostly use the web to click generate and load report. I think we can leave both as an option.

Reply:
Interesting.. i never use the web to generate and have the rake report generation running hourly in a heroku scheduler.

Reply:
ha interesting to see different use cases, when I started to add the various dynamic web options I have basically stopped using the rake tasks. I think it is worth maintaining both interfaces though. Rake tasks also integrate well into folks CI workflows.